src/core/panic.c

```diff
@@ -20,7 +20,7 @@
 #include "core/nng_impl.h"
 
 // Panic handling.
-void
+static void
 nni_show_backtrace(void)
```

Comment:
This was external for a reason -- the nni_ prefix indicates symbols that are meant to be available for external linkage, but are not documented for use by other projects.  In particular I sometimes use this function during debugging.

Reply:
That's o.k. But then I would still expect a function src/core/panic.h
I can change this.